src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1164,3 +1164,8 @@ Glyphicon for success status messages
     padding-right: 2px;
     content: '\e084';
 }
+
+.remind-no-response {
+    display: inline-block;
+    padding-right: 5px;
+}
```

Comment:
Should probably put this in the "main content" section of the file. (the very bottom of the file appears to be regarding responsiveness)